examples/demo/demo.rs

```diff
@@ -26,25 +28,66 @@ fn main() {
 }
 
 fn startup(mut world: Mut<World>, mut assets: Mut<Assets>, mut input: Mut<Input>) {
-    use cgmath::Rotation3;
-    assets.import("assets/crate.png", "crate");
+    assets.import("assets/crate.png");
 
     let texture = assets.register::<Texture>("crate");
     let cube1 = assets.store::<Mesh>(Mesh::cube(), "cube1");
     let cube2 = assets.store::<Mesh>(Mesh::cube(), "cube2");
-    let transform = Transform {
-        scale: Some(cgmath::Vector3::<f32>::new(2.0, 0.5, 0.8)),
-        translate: Some(cgmath::Vector3::<f32>::new(3.5, 0.0, 1.0)),
-        rotate: Some(cgmath::Quaternion::from_angle_y(cgmath::Rad(std::f32::consts::PI / 4.0))),
+    let transform1 = Transform {
+        scale: cgmath::Vector3::<f32>::new(0.5, 0.5, 0.5),
```

Comment:
Will like to have some "easier" way for this in future. Something like `Vector3::new()`.

About Transform
Do we need `Option` for translate, rotate and scale?
What about renaming `translate` to `position`?

Reply:
> Will like to have some "easier" way for this in future. Something like Vector3::new().
If you add `use cgmath::Vector3` you can write `translate: Vector3::new(...)`
I would suggest to add some generic constructors like `Transform::from_translate(x, y, z)`, `Transform::from_rotate(x, y, z)`, `Transform::from_scale(factor)` etc

> Do we need Option for translate, rotate and scale?
`Option` was there because of the way, how animations are stored in GLTF. I had to differentiate unset state and any other value. These functionality was moved into `TransformBuilder` structure. Because for Model transformation it overcomplicates things.


> What about renaming translate to position?
Well it is kind of standard for transformation: translation, rotation and scaling. And it is relative to default position of the model. For example, if model designed to be located at (5.0, 0.0, 3.0) and you move it by (2.0, 0.0, -1.0) result will be (7.0, 0.0, 2.0). So I would keep it as is. I think we will have position as `position in world` component

> What about having Transform as Component?
It is possible only if we make `Transform` to be mandatory used with `Model`, because that value is required for shaders. Otherwise Model won't be rendered.